docs/source/conf.py

```diff
@@ -43,8 +43,13 @@
     'sphinx.ext.todo',
     'sphinx.ext.coverage',
     'sphinx.ext.imgmath',
-    'sphinx.ext.githubpages',
-]
+    'sphinx.ext.githubpages'#,
+    #'sphinx.ext.mathjax'
+    ]
+
+
+#mathjax_path = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/MathJax.js?config=TeX-MML-AM_CHTML"
```

Comment:
@adamhb did you manage to get mathjax to build the doc on your local machine?

Reply:
Yes! Worked well.

Reply:
Excellent!  This will help us get that working in the official documentation.

Reply:
@glemieux sorry, actually mathjax didn't work for me and I ended up using the imgmath package instead. I believe the reason it didn't work was because of some package dependencies I didn't have on my linux machine, so it still could be easy to get this going with the proper dependencies. The code I have in conf.py file should still be correct though assuming the other dependencies are installed.